trino-api/Makefile

```diff
@@ -0,0 +1,75 @@
+DOCKER_DIR := "./deployment/dev"
```

Comment:
this Makefile wont be needed once the module is merged to main module of this repo.